_posts/2020-06-25-new-release-2020-3(Beta).md

```diff
@@ -0,0 +1,21 @@
+---
+title:  "Tableau 2020.3 Beta released"
+abstract: "Sneak peek on the features available on 2020.3"
+---
+
+With the release of Tableau 2020.3 Beta, you get a sneak peek into the features available in Tableau 2020.3 and connector SDK. 
+- __Connection Dialogs v2__ <br/>
+Connection Dialogs v2 feature, which will allow for more flexible connection dialogs. Learn more about connection Dialogs v2 [here]({{ site.baseurl }}/docs/mcd).
+- __Updated Dialect Samples__ <br/>
+The 2020-3 branch of connector SDK contains the complete dialect definition file for [Postgres JDBC connector](https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.3/samples/plugins/postgres_jdbc/dialect.tdd) and [Postgres ODBC connector](https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.3/samples/plugins/postgres_odbc/dialect.tdd). This should serve as a good example to write a dialect file for JDBC or ODBC connectors with 2020.3 updates.
+- __Delegation Delegation UID__ <br/>
+This applies to databases that support a user(authenticated_user) that can delegate requests to another user(delegated_user) by enabling the client to pass a DelegationUID(delegated_user) to the database server. More details on this feature will be added with the release of 2020.3. 
+- __Scenario-based Samples__ <br/>
+Scenario-based samples provide good examples for specific scenario partners might face while developing their connectors.
+    - __Extract Only Connector__ See more [here](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios/extract_only/sqlite_extract).
+    - __MySQL Based Connector__ See more [here](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios/mysql_based/mysql_odbc).
+    - __Connectors Using Vendor Attributes__ See more [here](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios/vendor_attributes/postgres_vendor).
+- __'format-true' and 'format-false' to allow literal, predicate and value attributes__ <br/>
+With the release of Tableau 2020.3, `format-true` and `format-false` will have two parameters, literal and predicate. <br/> See more details in the *Boolean Support* section of [dialect documentation]({{ site.baseurl }}/docs/dialect).
+
+
```

Comment:
Also add a section for format-string-literal's new unicode-prefix option:

With the release of Tableau 2020.3, `format-string-literal` has a new paramter: `unicode-prefix`.

Will add documentation for that today, think there's a  story somewhere but it'll be nice and quick.